intro-HPC/ch_web_portal.tex

```diff
@@ -0,0 +1,217 @@
+to your VSC account.
+
+Through the web portal, you can:
+\begin{itemize}
+    \item browse through your files \& directories, and inspect, manage or change them;
+    \item consult active jobs (across all HPC-UGent Tier-2 clusters);
+    \item submit new jobs to the HPC-UGent Tier-2 clusters, either from existing job scripts or from job templates;
+    \item start an interactive graphical user environment (a.k.a. desktop), either on the login nodes or on a cluster workernode;
+    \item open a terminal session directly in your web browser;
+\end{itemize}
+
+More detailed information is available below, as well as in the \href{https://osc.github.io/ood-documentation/master/}{Open OnDemand documentation}.
+
+An Open OnDemand walkthrough video is available on YouTube \href{https://www.youtube.com/watch?v=4-w-4wjlnPk}{here}.
+
+\section*{Pilot access}
+
+\strong{The HPC-UGent web portal is currently available for all VSC accounts,
+but there may be a couple of small problems that we still need to resolve.}
+
+\strong{If you notice anything that doesn't work as expected or could be improved,
+please contact \hpcinfo.}
+
+\strong{In addition, you \emph{must} be connected to the HPC-UGent network
+(via \href{https://helpdesk.ugent.be/vpn/en/}{VPN} if necessary)
+in order to use the HPC-UGent web portal.}
+
+\section{Login}
+
+When visiting the HPC-UGent web portal via \url{https://login.hpc.ugent.be},
+you will be automatically logged in via the VSC accountpage, which wll in turn
+authenticate you via your institutional login service (for UGent also known as CAS).
+
+You may need to enter your UGent login name and password in the standard CAS login page if you are not logged in yet.
+
+\subsection{First login}
+
+The first time you visit \url{https://login.hpc.ugent.be} permission will be requested to let Open OnDemand
+access the VSC accountpage to authenticate you, as shown in this screenshot below:
+
+\begin{center}
+    \includegraphics*[scale=0.5]{ood_permission}
+\end{center}
+
+\strong{Please click "Authorize" here.}
+
+This request will only be made once, you should not see this again afterwards.
+
+\section{Start page}
+
+Once logged in, you should see this start page:
+
+\begin{center}
+    \includegraphics*[scale=0.4]{ood_start}
+\end{center}
+
+This page includes a menu bar at the top, with buttons on the left providing access to the different features supported by Open OnDemand,
+as well as a \emph{Help} menu, your VSC account name, and a \emph{Log Out} button on the top right,
+and the familiar HPC-UGent welcome message with a high-level overview of the HPC-UGent Tier-2 clusters.
+
+If your browser window is too narrow, the menu is available at the top right through the ``hamburger'' icon:
+
+\begin{center}
+    \includegraphics*[scale=0.4]{ood_hamburger}
+\end{center}
+
+
+\section{Features}
+
+We briefly cover the different features provided by the web portal, going from left to right (or top to bottom) in the menu.
```

Comment:
`or top to bottom` is only relevant for the drop down menus?

Reply:
Yeah indeed, so removed.